tajo-catalog/tajo-catalog-common/src/main/proto/CatalogProtos.proto

```diff
@@ -247,13 +240,15 @@ message PartitionDescProto {
   required string partitionName = 1;
   repeated PartitionKeyProto partitionKeys = 2;
   optional string path = 3;
```

Comment:
Path also looks mandatory field. Is there any case that path is null? 
If not, it would be great if you also change it to ```required```.